src/ResourceManager/Consumption/Commands.Consumption.Test/ScenarioTests/BudgetTests.ps1

```diff
@@ -135,6 +147,9 @@ function Test-BudgetAtResourceGroupLevel
 	Assert-AreEqual True $response
 
 	Assert-Throws {Get-AzureRmConsumptionBudget -Name $budgetName -ResourceGroupName $resourceGroupName}
+
+	# Remove Resource Group
+	Remove-AzureRmResourceGroup -Name $resourceGroupName -Force
```

Comment:
@bgsky I would recommend wrapping the test (everything from `# Create Budget` to the command before this one) in a `try` block, and then the `finally` block would have this `Remove-AzureRmResourceGroup` call -- this ensures all resources created during this test are deleted if the test succeeds or fails